Browser/keywords/playwright_state.py

```diff
@@ -343,7 +343,8 @@ def new_persistent_context(
         forcedColors: ForcedColors = ForcedColors.none,
         url: Optional[str] = None,
     ):
-        """Open a new [persistent context | https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context].
+        """Open a new
+        [https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context | persistent context].
```

Comment:
Btw, is this really the right way of referencing URLs in a REST docstring?

Haven't tested it yet, but seems odd to me as I would expect:
```
`persistent context <https://playwright.dev/docs/api/class-browsertype#browser-type-launch-persistent-context>`_
```

Reply:
Yes, it's Robot Framework specific syntax.